.github/workflows/self-hosted-ci.yml

```diff
@@ -64,12 +64,15 @@ jobs:
         source compiler_modules
         export RRTMGP_ROOT=$PWD
         export FC=ftn
+        mkdir -p $PWD/tmp
```

Comment:
@alexeedm Can you explain how this temporary directory is used?

Reply:
NVHPC compiler uses temporary folder for compilation. On daint it ran out of space in the default tmp/ folder multiple times, according to the CI logs